homeassistant/components/google/calendar.py

```diff
@@ -89,14 +89,25 @@ def _async_setup_entities(
 ) -> None:
     calendar_service = hass.data[DOMAIN][DATA_SERVICE]
     entities = []
+    num_entities = len(disc_info[CONF_ENTITIES])
     for data in disc_info[CONF_ENTITIES]:
-        if not data[CONF_TRACK]:
-            continue
-        entity_id = generate_entity_id(
-            ENTITY_ID_FORMAT, data[CONF_DEVICE_ID], hass=hass
-        )
+        entity_enabled = data[CONF_TRACK]
```

Comment:
I was doing some additional manual testing for backwards compatibility and noticed that once this is loaded, then changing the google_calendars.yaml to set `track` to `False` for a calendar did not have any effect. It seems like once `attr_entity_registry_enabled_default` is set then it won't change.

My current thought is to leave it that way and update the documentation to encourage use of disabling/enabling the entities in the UI instead, but open to feedback.

Reply:
I'd remove this feature from the YAML config. If it doesn't work as expected it's a better user experience to not offer it.

Reply:
Let me try to clarify the plan to make sure this makes sense:
- Here, import the existing `track` state into `attr_entity_registry_enabled_default`. My impression is that since we're adding unique ids this is able to set the initial state one time and it correctly reflects the user intent.  (However, as a user updating`track` in yaml no longer does anything)
- Mark `track`  as deprecated in the device schema
- Stop setting `track` when updating `google_calendars.yaml` going forward
- Remove it from the documentation

Reply:
Yes, that's sounds good!